sqrl-server/sqrl-server-core/src/main/java/com/datasqrl/graphql/server/GraphQLEngineBuilder.java

```diff
@@ -110,9 +110,9 @@ public GraphQL.Builder visitRoot(RootGraphqlModel root, Context context) {
 
     if (root.subscriptions != null) {
       for (SubscriptionCoords sc : root.subscriptions) {
+        DataFetcher<?> subscriptionFetcher = sc.accept(context.createSubscriptionFetcherVisitor());
```

Comment:
It might be cleaner to do pass a mutation/subscription factory into GraphQLEngineBuilder rather than shuttle it through the context. I was using the context as a way to keep jdbc connection details and using it also for object construction was probably a mistake on my part.